acceptance/suites/tests/010-puppetserver-cli/subcommand/foreground.rb

```diff
@@ -0,0 +1,44 @@
+test_name "Puppetserver 'foreground' subcommand tests."
+
+# --------------------------------------------------------------------------
+# This behavior needs to be baked into Beaker but is not yet. SERVER-79 tracks
+# the need to fix this once Beaker has been modified to make the paths to these
+# commands available.
+#
+if master.is_pe?
+  cli = "pe-puppetserver"
+else
+  cli = "puppetserver"
+end
+# --------------------------------------------------------------------------
+
+# puppetserver seems to take about 45s to start up
+timout_length = "60s"
+foreground_cmd = "#{cli} foreground"
+timeout_cmd = "timeout -s INT #{timout_length} #{foreground_cmd}"
+
+expected_messages = {
+  /Initializing the JRuby service/ => "JRuby didn't initialize",
+  /Starting web server/ => "Expected web server to start",
+  /Finished creating JRubyPuppet instance 4 of 4/ => "Expected to find 4 JRubyPuppet instances",
```

Comment:
I don't think we explicitly set a value for the number of JRubies in the acceptance test runs, so it'd probably be best to avoid hard-coding the `4`s into the test here.

I think there's a log message that Preben added that says something like "Puppet Server is now ready to handle requests".  That might be a good replacement for the test.